doc/nrf/releases_and_maturity/migration/migration_guide_2.8.rst

```diff
@@ -53,12 +201,21 @@ Serial LTE Modem (SLM)
     * ``AT#XSOCKETOPT=1,53,`` with ``AT#XSOCKETOPT=1,61,4`` to indicate ``RAI_ONGOING``.
     * ``AT#XSOCKETOPT=1,54,`` with ``AT#XSOCKETOPT=1,61,5`` to indicate ``RAI_WAIT_MORE``.
 
-SUIT DFU for nRF54H20
----------------------
+.. _migration_2.8_required_nrf54h:
+
+nRF54H20
+--------
+
+.. toggle::
+
+  * When using the nRF54H20 DK Engineering B (from v0.8.0 to 0.8.2), you must build samples and applications using the ``nrf54h20dk@0.8.0/nrf54h20/cpuapp`` build target.
+    See the following example::
```

Comment:
Agreeing with Robert, I think this would work better as "For example, to build the :ref:`tag` sample, use the following command.